mail-client/mutt-wizard/mutt-wizard-20200430.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+DESCRIPTION="A system for automatically configuring mutt and isync"
+HOMEPAGE="https://github.com/LukeSmithxyz/mutt-wizard"
+
+COMMIT=6fe26400957d9163a75f60fa0c18f266f25fbc85
+SRC_URI="https://github.com/LukeSmithxyz/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+LICENSE="GPL-3"
+SLOT="0"
+
+RDEPEND="
+	mail-client/neomutt:=[notmuch]
+	net-mail/isync:=[ssl]
```

Comment:
Since this does no compiling, no libraries can be linked. (Take out `:=`)

Reply:
Done.